src/App.jsx

```diff
@@ -1,104 +1,26 @@
 import './App.scss';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostsList } from './components/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId);
+}
+
+function getCommentsById(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId);
+}
+
+const postArray = postsFromServer.map(post => ({
+  ...post,
+  user: getUserById(post.userId),
+  comments: getCommentsById(post.id) || `No comments yet`,
```

Comment:
The fallback `|| 'No comments yet'` is unnecessary because `getCommentsById` will always return an array, even if it's empty. You can remove this fallback to simplify the code.